doc/reference/configuration/configuration_reference.rst

```diff
@@ -2174,3 +2174,340 @@ The ``security`` section defines configuration parameters related to various sec
     | Environment variable: TT_SECURITY_SECURE_ERASING
+
+..  _configuration_reference_snapshot:
+
+snapshot
+--------
+
+The ``snapshot`` section defines configuration parameters related to the :ref:`snapshot files <internals-snapshot>`.
+To learn more about the snapshots' configuration, check the :ref:`Persistence <configuration_persistence_snapshot>` page.
+
+..  NOTE::
+
+    ``snapshot`` can be defined in any :ref:`scope <configuration_scopes>`.
+
+-   :ref:`snapshot.dir <configuration_reference_snapshot_dir>`
+-   :ref:`snapshot.snap_io_rate_limit <configuration_reference_snapshot_snap_io_rate_limit>`
+-   :ref:`snapshot.count <configuration_reference_snapshot_count>`
+-   :ref:`snapshot.by.* <configuration_reference_snapshot_by>`
+
+    -   :ref:`snapshot.by.interval <configuration_reference_snapshot_by_interval>`
+    -   :ref:`snapshot.by.wal_size <configuration_reference_snapshot_by_wal_size>`
+
+..  _configuration_reference_snapshot_dir:
+
+..  confval:: snapshot.dir
+
+    A directory where memtx stores snapshot (.snap) files. Can be relative to
+    ``process.work_dir``. If not specified, defaults to
+    ``process.work_dir``. See also: :ref:`wal.dir <configuration_reference_wal_dir>`.
+
+    |
+    | Type: string
+    | Default: 'var/lib/{{ instance_name }}'
+    | Environment variable: TT_SNAPSHOT_DIR
+
+..  _configuration_reference_snapshot_snap_io_rate_limit:
+
+..  confval:: snapshot.snap_io_rate_limit
+
+    Reduce the throttling effect of :doc:`box.snapshot() </reference/reference_lua/box_snapshot>` on
+    INSERT/UPDATE/DELETE performance by setting a limit on how many
+    megabytes per second it can write to disk. The same can be
+    achieved by splitting :ref:`wal.dir <configuration_reference_wal_dir>` and
+    :ref:`snapshot.dir <configuration_reference_snapshot_dir>`
+    locations and moving snapshots to a separate disk.
+    The limit also affects what
+    :ref:`box.stat.vinyl().regulator <box_introspection-box_stat_vinyl_regulator>`
+    may show for the write rate of dumps to ``.run`` and ``.index`` files.
+
+    |
+    | Type: number
+    | Default: box.NULL
+    | Environment variable: TT_SNAPSHOT_SNAP_IO_RATE_LIMIT
+
+..  _configuration_reference_snapshot_count:
+
+..  confval:: snapshot.count
+
+    The maximum number of snapshots that may exist on the
+    :ref:`snapshot.dir <configuration_reference_snapshot_dir>` directory
+    before the :ref:`checkpoint daemon <configuration_persistence_checkpoint_daemon>` deletes old snapshots.
+    If ``snapshot.count`` equals zero, then the checkpoint daemon
+    does not delete old snapshots.
+
+    Example:
+
+    ..  code-block:: yaml
+
+        snapshot:
+          by:
+            interval: 60
+          count: 10
+
+    In the example, the checkpoint daemon creates a new snapshot each hour until
+    it has created ten snapshots. After that, it deletes the oldest snapshot
+    (and any associated write-ahead-log files) after creating a new one.
```

Comment:
All the time interval options are in seconds. It means that `snapshot.by.interval = 60` is one minute, not one hour. One hour is `3600`.